content/tutorials/quickstart/batch-minting.md

```diff
@@ -0,0 +1,494 @@
+---
+html: batch-minting.html
+parent: xrpl-quickstart.html
+blurb: Broker a sale between a sell offer and a buy offer.
+labels:
+  - Accounts
+  - Quickstart
+  - Broker
+  - XRP
+---
+
+# Batch Mint NFTokens
+
+You can create an application that mints multiple NFTokens at one time. You can use a `for` loop to send one transaction after another.
+
+A best practice is to use `Tickets` to reserve the transaction sequence numbers. If you create an application that creates NFTokens without using tickets, if any transaction fails for any reason, the application stops with an error. If you use tickets, the application continues to send transactions, and you can look into the reason for the failure afterward.
+
+## Usage
+
+You can download the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/quickstart.zip) archive to try the sample in your own browser.
+
+## Get an Account
+
+1. Open `7.batch-minting.html` in a browser.
+2. Get a test account.
+    1. If you want to use an existing NFT-Devnet account seed:
+        1. Paste the account seed in the **Seed** field.
+        2. Click **Get Account from Seed**.
+    2. If you do not want to use an existing NFT-Devnet account seed, click **Get New Standby Account**.
+
+**Note:** Running this command throws an error in the JavaScript console because the `getAccountsFromSeeds` function in `ripplex1-send-xrp.js` looks for the operational seed field, which is not included in this form. You can ignore the error (or fix it in your own implementation).
+
+## Batch Mint NFTokens
+
+This example lets you mint multiple NFTokens for a single unique item. The NFToken might represent "prints" of an original artwork, tickets to an event, or another limited set of unique items. 
+
+To batch mint a non-fungible token objects:
+
+1. Set the **Flags** field. For testing purposes, we recommend setting the value to _8_. This sets the _tsTransferable_ flag, meaning that the NFToken object can be transferred to another account. Otherwise, the NFToken object can only be transferred back to the issuing account. See [NFToken Mint](https://xrpl.org/nftokenmint.html#:~:text=Example%20NFTokenMint%20JSON-,NFTokenMint%20Fields,-NFTokenMint%20Flags) for information about all of the available flags for minting NFTokens.
```

Comment:
Use a relative link here rather than a fully-qualified URL so that it doesn't get marked as an "external" link. Also, that `#:~:text=...` anchor doesn't seem to do anything, so I removed it.

```suggestion
1. Set the **Flags** field. For testing purposes, we recommend setting the value to _8_. This sets the _tsTransferable_ flag, meaning that the NFToken object can be transferred to another account. Otherwise, the NFToken object can only be transferred back to the issuing account. See [NFTokenMint](nftokenmint.html) for information about all of the available flags for minting NFTokens.
```